ReplyAdaptabilityCodelab/app/src/main/java/com/example/reply/data/AccountsRepositoryImpl.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.reply.data
+
+import com.example.reply.data.local.LocalAccountsDataProvider
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class AccountsRepositoryImpl : AccountsRepository {
```

Comment:
Do we need a comment indicating this is a mock (hence the `delay`s)? Same with the other repositories

Reply:
Sure let me add comments

Reply:
Just that this file is never used. Let's remove it together with the interface for it.